src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,7 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Ball randomBall = new Lottery().getRandomBall();
```

Comment:
Don't create several instances of the same class if you can use only one instance for your purposes. In this case, you should create a single `Lottery` instance and use it to get the `randomBall`.